components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.geo-dashboard/public/js/websocket.js

```diff
@@ -518,26 +561,20 @@ var webSocketOnAlertMessage = function processMessage(message) {
         var json = $.parseJSON(message.data);
         if (json.messageType == "Alert") {
             processAlertMessage(json);
-        } else {
+        }else {
```

Comment:
@lasanthaDLPDS Please fix the formatting issue here.

Reply:
This is fixed c86c976fe6ca2d0e8b534b7efc441713c7e529b7.